mw-webapp/src/component/text/Text.tsx

```diff
@@ -0,0 +1,60 @@
+import clsx from "clsx";
+import styles from "src/component/text/Text.module.scss";
+
+/**
+ * Type of  text variant styles
+ */
+
+export type ThemeVariant = "clear" | "outline";
```

Comment:
1. let's use enum
2. Let's rename to TextType (see example in Button component) instead of ThemeVariant
3. Let's use only one value right now because design is not ready, for example "default"

Reply:
Replaced the type with an enum and left the default type